Makefile

```diff
@@ -47,13 +47,21 @@ build-chalice-config:
 	cd .chalice; jq .stages.$(STAGE).tags.service=env.APP_NAME config.json | sponge config.json
 	cd .chalice; jq .stages.$(STAGE).tags.owner=env.OWNER config.json | sponge config.json
 
+# package prepares a Lambda zipfile with the help of the Chalice packager (which also emits a SAM template).
+# We also inject any wheels found in vendor.in, and rewrite the zipfile to make the build reproducible.
 package:
 	rm -rf vendor
 	mkdir vendor
 	cp -a $(APP_NAME) $(APP_NAME)-api.yml vendor
+	find vendor -name '*.pyc' -delete
+	find vendor -exec touch -t 201901010000 {} \;
```

Comment:
can you add a comment about what this line is doing/why it is necessary

Reply:
Will do.

Reply:
Done.